harvester-vm-dhcp-controller/harvester-vm-dhcp-controller.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: harvester-system
+---
```

Comment:
namespace is not needed. For rancher-vcluster we do create one because it needs to be in non `harvester-system` namespace.

Reply:
Thanks for the heads-up!